Makefile

```diff
@@ -237,6 +237,26 @@ generate_mocks: ## TODO: auto install go install go.uber.org/mock/mockgen@latest
 	mkdir -p ${GEN_DIR}
 	mockgen -destination ${GEN_DIR}/${NETBOX_MOCKS_OUTPUT_FILE} -source=${INTERFACE_DEFITIONS_DIR}
 
-.PHONY: test-e2e
-test-e2e: install-$(GO_PACKAGE_NAME_CHAINSAW)
-	chainsaw test --namespace e2e
+# e2e tests
+# setting parallel to 2 to avoid flakiness
+E2E_PARAM := --namespace e2e --parallel 3 --apply-timeout 3m --assert-timeout 3m --delete-timeout 3m --error-timeout 3m --exec-timeout 3m --skip-delete
```

Comment:
Setting to 2 but actually set to 3 - which is it supposed to be?

Reply:
I will remove the comment :D 
Good catch!